src/cartridges/int_adyen_overlay/cartridge/scripts/handleCustomObject.js

```diff
@@ -71,7 +71,9 @@ function handle(customObj) {
 
   // split order ID by - and remove last split (which is the date)
   const orderIdSplit = customObj.custom.orderId.split('-').slice(0, -1);
```

Comment:
```suggestion
  const orderIdParts = customObj.custom.orderId.split('-');
  orderIdParts.pop();
```
slice creates a new array without the last element. pop uses same one